src/main/java/com/scrive/pdftools/tools/utils/FileInput.java

```diff
@@ -0,0 +1,42 @@
+package com.scrive.pdftools.tools.utils;
+
+import com.itextpdf.text.pdf.codec.Base64;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+
+public class FileInput {
+    private String localFilePath;
+    private String base64Content;
+    private byte[] content;
+
+    private boolean isValid() {
+        return this.localFilePath != null || this.base64Content != null;
+    }
+    public byte[] getContent() throws IOException {
+        if (this.content != null ) {
+            return content;
+        } else if (this.localFilePath != null && this.localFilePath != "") {
+            RandomAccessFile f = new RandomAccessFile(this.localFilePath , "r");
+            this.content = new byte[(int)f.length()];
```

Comment:
but even better would be replacing this with the usage of `IOUtils`.
`this.content = IOUtils.toByteArray(f)`